calendar/screen.css

```diff
@@ -333,6 +333,11 @@ input:checked + label {
   padding: 0px;
 }
 
+.toastui-calendar-week-view .toastui-calendar-holiday-sun,
```

Comment:
why not use theme property instead: 
https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/theme.md#common-holiday ?

Reply:
It applies the accent to more places than these changes, which isn't desirable (e.g. the month view).